tests/integration/bond_test.py

```diff
@@ -99,6 +100,16 @@ def bond99_with_slave(eth2_up):
         yield state
 
 
+def test_remove_existing_bond(setup_remove_bond99):
+    subprocess.check_call(["ip", "link", "add", BOND99, "type", "bond"])
+    assertlib.assert_present(BOND99, timeout=1)
+
+    with bond_interface(BOND99, [], create=False):
```

Comment:
This does nothing except creating the state (`create=False` means it is not applied)

Reply:
AFAIU it removes the bond when the context is left, does it not?

Reply:
Right.